tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/regression.py

```diff
@@ -97,6 +97,21 @@ def process(self, raw, identifiers, frame_meta):
         return result
 
 
+class OnlineMultiOutputRegression(MultiOutputRegression):
```

Comment:
I think it would be better to update the existing `MultiOutputRegression` adapter to avoid code duplication. You can add an additional optional parameter and, depending on its value, save in `res_dict` either `raw_outputs[output_name_v]` or `raw_outputs[output_name_v][batch_id]`